content/03.when-to-use-dl.md

```diff
@@ -0,0 +1,51 @@
+## Tip 1: Decide whether your problem is appropriate for deep learning {#appropriate}
+
+In recent years, the number of publications implementing DL in biology have risen tremendously.
+Given DL's usefulness across a range of scientific questions and data modalities, it may appear that it is a panacea for modeling problems.
+Indeed, neural networks are universal function approximators, meaning that they are in principle capable of learning any function [@doi:10.1007/BF02551274; @tag:hornik-approximation].
+If DL is so powerful and popular, why would one ever not choose to use it?
+
+The reason is simple: DL is not suited to every situation in reality.
+Training DL models requires a significant amount of data, computing power, and expertise.
+In some areas of biology where data collection is thoroughly automated, such as DNA sequencing, large amounts of quality data may be available.
+For other areas which rely on manual data collection, there may not be enough data to effectively train models.
+Though there are methods to increase the amount of training data such as data augmentation (in which existing data is slightly manipulated to yield "new" samples) and weak supervision (in which simple labeling heuristics are combined to produce noisy, probabilistic labels) [@arxiv:1605.07723v3], these methods cannot overcome a complete shortage of data.
+As a rule of thumb, DL should only be considered for datasets with at least one thousand samples, though it is best suited to cases when datasets contain orders of magnitude more samples.
```

Comment:
Should have a reference for this rule of thumb.

Reply:
@beamandrew can you help out with this sentence? I see you have a blog post on the topic: https://beamandrew.github.io/deeplearning/2017/06/04/deep_learning_works.html

Reply:
We can probably revise the number down to ~100 and cite this: https://arxiv.org/abs/1511.06348

We should also mention one-shot learning, but saying that you require a minimum of one data point in your training data is somewhat useless for the beginner deciding whether to use DL.

Reply:
Ok I added the citation and revised it down.